src/api/libcellml/analyser.h

```diff
@@ -58,6 +58,171 @@ class LIBCELLML_EXPORT Analyser: public Logger
      */
     void analyseModel(const ModelPtr &model);
 
+    /**
+     * @brief Add an @c AnalyserExternalVariable to this @c Analyser.
+     *
+     * Add the given @c AnalyserExternalVariable to this @c Analyser, but only
+     * if the given @c AnalyserExternalVariable or its primary equivalent has
+     * not already been added.
```

Comment:
It wasn't explained, but you in fact spotted a mistake in the documentation. We check against the primary equivalent variable during the analysis, not when adding an external variable. So, I have updated the description to remove "or its primary equivalent". I have also updated the documentation for `AnalyserVariable::variable()` since it is the method that returns what I call the primary variable. (So, primary equivalent variable is a primary variable that is equivalent to something.) All done in commit a9731860a.